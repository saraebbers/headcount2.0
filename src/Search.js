import React, { Component } from 'react';

class Search extends Component {
  constructor() {
    super();
    this.state = {

    }
  }

  render() {
    return(
      <div>
        <h3>Enter the district you want to search</h3>
        <form>
          <input />
          <button>Submit</button>
        </form>
      </div>
    )
  }
}

export default Search;