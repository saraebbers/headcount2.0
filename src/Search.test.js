import React from 'react';
import Search from './Search';
import { shallow } from 'enzyme';

describe('Search', () => {
  it('matches the snapshot', () => {
    const wrapper = shallow(<Search searchForDistrict={()=> {}} />);
    expect(wrapper).toMatchSnapshot()
  });


})