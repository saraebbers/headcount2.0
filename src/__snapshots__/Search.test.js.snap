// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search matches the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Search
    searchForDistrict={[Function]}
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          Enter the district you want to search
</h3>,
        <form
          onSubmit={[Function]}
>
          <input
                    className="inputField"
                    name="searchString"
                    onChange={[Function]}
                    placeholder="Enter District You Want To Search"
                    value=""
          />
          <button
                    className="inputButton"
          >
                    Submit
          </button>
</form>,
      ],
      "className": "search",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Enter the district you want to search",
        },
        "ref": null,
        "rendered": "Enter the district you want to search",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <input
              className="inputField"
              name="searchString"
              onChange={[Function]}
              placeholder="Enter District You Want To Search"
              value=""
/>,
            <button
              className="inputButton"
>
              Submit
</button>,
          ],
          "onSubmit": [Function],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "inputField",
              "name": "searchString",
              "onChange": [Function],
              "placeholder": "Enter District You Want To Search",
              "value": "",
            },
            "ref": null,
            "rendered": null,
            "type": "input",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Submit",
              "className": "inputButton",
            },
            "ref": null,
            "rendered": "Submit",
            "type": "button",
          },
        ],
        "type": "form",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            Enter the district you want to search
</h3>,
          <form
            onSubmit={[Function]}
>
            <input
                        className="inputField"
                        name="searchString"
                        onChange={[Function]}
                        placeholder="Enter District You Want To Search"
                        value=""
            />
            <button
                        className="inputButton"
            >
                        Submit
            </button>
</form>,
        ],
        "className": "search",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Enter the district you want to search",
          },
          "ref": null,
          "rendered": "Enter the district you want to search",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <input
                className="inputField"
                name="searchString"
                onChange={[Function]}
                placeholder="Enter District You Want To Search"
                value=""
/>,
              <button
                className="inputButton"
>
                Submit
</button>,
            ],
            "onSubmit": [Function],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "inputField",
                "name": "searchString",
                "onChange": [Function],
                "placeholder": "Enter District You Want To Search",
                "value": "",
              },
              "ref": null,
              "rendered": null,
              "type": "input",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Submit",
                "className": "inputButton",
              },
              "ref": null,
              "rendered": "Submit",
              "type": "button",
            },
          ],
          "type": "form",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
